docs/endpoint.md

```diff
@@ -3,21 +3,23 @@
 This page details the usage of the endpoint command
 
 ```
-Usage: hugie endpoint[OPTIONS] COMMAND[ARGS]...
-
-Options:
-  --help  Show this message and exit.
-
-Commands:
-  create  Create an endpoint
-  delete  Delete an endpoint
-  info    Get info about an endpoint
-  list    List all the deployed endpoints
-  logs    Get logs about an endpoint
-  test    Test and endpoint
-  update  Update an endpoint
+ Usage: hugie endpoint [OPTIONS] COMMAND [ARGS]...
+
+╭─ Options ────────────────────────────────────────────────────────────────────╮
+│ --help          Show this message and exit.                                  │
+╰──────────────────────────────────────────────────────────────────────────────╯
+╭─ Commands ───────────────────────────────────────────────────────────────────╮
+│ create         Create an endpoint                                            │
+│ delete         Delete an endpoint                                            │
+│ info           Get info about an endpoint                                    │
+│ list           List all the deployed endpoints                               │
+│ logs           Get logs about an endpoint                                    │
+│ test           Test an endpoint                                              │
+│ update         Update an endpoint                                            │
```

Comment:
shouldn't this contain `config`?

Reply:
No, it's a separate sub command, so it should be in docs/config